2021-02-26 14:10:01  [ localhost-startStop-1:0 ] - [ DEBUG ]  Logging initialized using 'class org.apache.ibatis.logging.slf4j.Slf4jImpl' adapter.
2021-02-26 14:10:01  [ localhost-startStop-1:240 ] - [ DEBUG ]  Registered plugin: 'com.github.pagehelper.PageInterceptor@7886220b'
2021-02-26 14:10:01  [ localhost-startStop-1:350 ] - [ DEBUG ]  Parsed mapper file: 'file [C:\Users\Administrator\IdeaProjects\ElectricityConsumption\target\classes\mapper\EleConDao.xml]'
2021-02-26 14:10:01  [ localhost-startStop-1:360 ] - [ DEBUG ]  Parsed mapper file: 'file [C:\Users\Administrator\IdeaProjects\ElectricityConsumption\target\classes\mapper\EleConTemDao.xml]'
2021-02-26 14:10:01  [ localhost-startStop-1:370 ] - [ DEBUG ]  Parsed mapper file: 'file [C:\Users\Administrator\IdeaProjects\ElectricityConsumption\target\classes\mapper\EleConWeibiaoDao.xml]'
2021-02-26 14:10:02  [ localhost-startStop-1:800 ] - [ DEBUG ]  Logging Provider: org.jboss.logging.Log4jLoggerProvider
2021-02-26 14:10:02  [ localhost-startStop-1:800 ] - [ INFO ]  HV000001: Hibernate Validator 5.1.3.Final
2021-02-26 14:10:02  [ localhost-startStop-1:810 ] - [ DEBUG ]  Found javax.persistence.Persistence on classpath, but no method 'getPersistenceUtil'. Assuming JPA 1 environment. All properties will per default be traversable.
2021-02-26 14:10:02  [ localhost-startStop-1:820 ] - [ DEBUG ]  Setting custom MessageInterpolator of type org.springframework.validation.beanvalidation.LocaleContextMessageInterpolator
2021-02-26 14:10:02  [ localhost-startStop-1:820 ] - [ DEBUG ]  Setting custom ConstraintValidatorFactory of type org.springframework.validation.beanvalidation.SpringConstraintValidatorFactory
2021-02-26 14:10:02  [ localhost-startStop-1:820 ] - [ DEBUG ]  Setting custom ParameterNameProvider of type org.springframework.validation.beanvalidation.LocalValidatorFactoryBean$1
2021-02-26 14:10:02  [ localhost-startStop-1:840 ] - [ DEBUG ]  Trying to load META-INF/validation.xml for XML based Validator configuration.
2021-02-26 14:10:02  [ localhost-startStop-1:850 ] - [ DEBUG ]  No META-INF/validation.xml found. Using annotation based configuration only.
2021-02-26 14:10:08  [ http-bio-8080-exec-2:7221 ] - [ INFO ]  SSM!!!!
2021-02-26 14:10:08  [ http-bio-8080-exec-2:7235 ] - [ DEBUG ]  Creating a new SqlSession
2021-02-26 14:10:08  [ http-bio-8080-exec-2:7244 ] - [ DEBUG ]  SqlSession [org.apache.ibatis.session.defaults.DefaultSqlSession@1acd7e1b] was not registered for synchronization because synchronization is not active
2021-02-26 14:10:08  [ http-bio-8080-exec-2:7368 ] - [ INFO ]  {dataSource-1} inited
2021-02-26 14:10:18  [ Druid-ConnectionPool-Create-1291540755:17400 ] - [ ERROR ]  create connection SQLException, url: jdbc:postgresql://25.60.253.231:8000/sjzt, errorCode 0, state 08001
org.postgresql.util.PSQLException: 尝试连线已失败。
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:257)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
	at org.postgresql.Driver.makeConnection(Driver.java:452)
	at org.postgresql.Driver.connect(Driver.java:254)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1513)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1578)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2466)
Caused by: java.net.SocketTimeoutException: connect timed out
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.postgresql.core.PGStream.<init>(PGStream.java:69)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:156)
	... 7 more
2021-02-26 14:10:28  [ Druid-ConnectionPool-Create-1291540755:27406 ] - [ ERROR ]  create connection SQLException, url: jdbc:postgresql://25.60.253.231:8000/sjzt, errorCode 0, state 08001
org.postgresql.util.PSQLException: 尝试连线已失败。
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:257)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
	at org.postgresql.Driver.makeConnection(Driver.java:452)
	at org.postgresql.Driver.connect(Driver.java:254)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1513)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1578)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2466)
Caused by: java.net.SocketTimeoutException: connect timed out
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.postgresql.core.PGStream.<init>(PGStream.java:69)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:156)
	... 7 more
2021-02-26 14:14:14  [ localhost-startStop-1:0 ] - [ DEBUG ]  Logging initialized using 'class org.apache.ibatis.logging.slf4j.Slf4jImpl' adapter.
2021-02-26 14:14:14  [ localhost-startStop-1:210 ] - [ DEBUG ]  Registered plugin: 'com.github.pagehelper.PageInterceptor@1c469bbf'
2021-02-26 14:14:14  [ localhost-startStop-1:310 ] - [ DEBUG ]  Parsed mapper file: 'file [C:\Users\Administrator\IdeaProjects\ElectricityConsumption\target\classes\mapper\EleConDao.xml]'
2021-02-26 14:14:14  [ localhost-startStop-1:330 ] - [ DEBUG ]  Parsed mapper file: 'file [C:\Users\Administrator\IdeaProjects\ElectricityConsumption\target\classes\mapper\EleConTemDao.xml]'
2021-02-26 14:14:14  [ localhost-startStop-1:340 ] - [ DEBUG ]  Parsed mapper file: 'file [C:\Users\Administrator\IdeaProjects\ElectricityConsumption\target\classes\mapper\EleConWeibiaoDao.xml]'
2021-02-26 14:14:15  [ localhost-startStop-1:808 ] - [ DEBUG ]  Logging Provider: org.jboss.logging.Log4jLoggerProvider
2021-02-26 14:14:15  [ localhost-startStop-1:808 ] - [ INFO ]  HV000001: Hibernate Validator 5.1.3.Final
2021-02-26 14:14:15  [ localhost-startStop-1:817 ] - [ DEBUG ]  Found javax.persistence.Persistence on classpath, but no method 'getPersistenceUtil'. Assuming JPA 1 environment. All properties will per default be traversable.
2021-02-26 14:14:15  [ localhost-startStop-1:826 ] - [ DEBUG ]  Setting custom MessageInterpolator of type org.springframework.validation.beanvalidation.LocaleContextMessageInterpolator
2021-02-26 14:14:15  [ localhost-startStop-1:826 ] - [ DEBUG ]  Setting custom ConstraintValidatorFactory of type org.springframework.validation.beanvalidation.SpringConstraintValidatorFactory
2021-02-26 14:14:15  [ localhost-startStop-1:827 ] - [ DEBUG ]  Setting custom ParameterNameProvider of type org.springframework.validation.beanvalidation.LocalValidatorFactoryBean$1
2021-02-26 14:14:15  [ localhost-startStop-1:854 ] - [ DEBUG ]  Trying to load META-INF/validation.xml for XML based Validator configuration.
2021-02-26 14:14:15  [ localhost-startStop-1:861 ] - [ DEBUG ]  No META-INF/validation.xml found. Using annotation based configuration only.
2021-02-26 14:14:18  [ http-bio-8080-exec-2:4401 ] - [ INFO ]  SSM!!!!
2021-02-26 14:14:18  [ http-bio-8080-exec-2:4413 ] - [ DEBUG ]  Creating a new SqlSession
2021-02-26 14:14:18  [ http-bio-8080-exec-2:4423 ] - [ DEBUG ]  SqlSession [org.apache.ibatis.session.defaults.DefaultSqlSession@6c318765] was not registered for synchronization because synchronization is not active
2021-02-26 14:14:19  [ http-bio-8080-exec-2:4539 ] - [ INFO ]  {dataSource-1} inited
